pylon-core/src/main/resources/lang/enws.yml

```
@@ -54,6 +61,63 @@ gui:
 
   time_left: "Time left: %time%"
 
+
+guide:
+  title: "Pylon Guide"
```

```suggestion
  title: "Pylon Bible (King James Version)"
```
Forgot to change this